bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -328,4 +328,26 @@
 
 		<config-description-ref uri="thing-type:avmfritz:fritzgroup"/>
 	</thing-type>
+
+	<thing-type id="FRITZ_DECT_500">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="fritzbox"/>
+			<bridge-type-ref id="FRITZ_Powerline_546E_Solo"/>
+		</supported-bridge-type-refs>
+
+		<label>FRITZ!DECT 500</label>
+		<description>FRITZ!DECT500 dimmable light.</description>
+
+		<channels>
+			<channel id="mode" typeId="mode"/>
+			<channel id="locked" typeId="locked"/>
+			<channel id="device_locked" typeId="device_locked"/>
+			<channel id="onoffstate" typeId="onoffstate"/>
```

Comment:
I think you can use a [system default channel type](https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types) for this Channel.

```suggestion
			<channel id="onoffstate" typeId="system.power"/>
```